public/theme/boltcms-2021/showcases.twig

```diff
@@ -0,0 +1,46 @@
+{% extends 'partials/_master.twig' %}
+
+{% block main %}
+    <body class="loading showcase">
+    <main>
+        <div data-scroll>
+            <div class="subheader">
+                <h1>{{ contenttype.name }}</h1>
+                <p>A fine selection of websites made with Bolt</p>
+            </div>
+            <div class="content">
+                {% for showcase in showcases %}
+                    <div class="item">
+                        {% if showcase.screenshot is not empty %}
+                            <div class="item__img-wrap"
+                                 style="--image: url({{ showcase.screenshot|thumbnail(700, 400) }});">
+                                <div class="item__img item__img--t1"></div>
+                            </div>
+                        {% endif %}
+                        <div class="item__caption">
+                            <h2 class="item__caption-title">{{ showcase.title }}</h2>
+                            <p class="item__caption-url"><a href="" target="_blank"
+                                                            rel="noopener noreferrer">{{ showcase.website_link }}</a>
+                            </p>
+                            <p class="item__caption-copy">{{ showcase.website_name }}</p>
+                            <p class="item__caption-url">Build by: <a href="" target="_blank" rel="noopener noreferrer">{{ showcase.contributor }}</a>
```

Comment:
If we can make the `contributor` linkable (see my previous comment), then we can also fix this empty `<a>` tag by removing it :-)

Reply:
ok